projects/extension/tests/vectorizer/test_vectorizer.py

```diff
@@ -260,37 +272,6 @@ def test_vectorizer_timescaledb():
             actual = cur.fetchone()[0]
             assert actual == 3
 
-            # bob should have select on the source table
-            cur.execute("select has_table_privilege('bob', 'website.blog', 'select')")
-            actual = cur.fetchone()[0]
-            assert actual
-
-            # bob should have select, update, delete on the queue table
-            cur.execute(
-                f"select has_table_privilege('bob', '{vec.queue_schema}.{vec.queue_table}', 'select, update, delete')"
-            )
-            actual = cur.fetchone()[0]
-            assert actual
-
-            # bob should have select, insert, update on the target table
-            cur.execute(
-                f"select has_table_privilege('bob', '{vec.target_schema}.{vec.target_table}', 'select, insert, update')"
-            )
-            actual = cur.fetchone()[0]
-            assert actual
-
-            # bob should have select on the view
-            cur.execute(
-                f"select has_table_privilege('bob', '{vec.view_schema}.{vec.view_name}', 'select')"
-            )
-            actual = cur.fetchone()[0]
-            assert actual
-
-            # bob should have select on the vectorizer table
-            cur.execute("select has_table_privilege('bob', 'ai.vectorizer', 'select')")
-            actual = cur.fetchone()[0]
-            assert actual
-
```

Comment:
why are we nuking these?

Reply:
those were added before we did specific validations under privileges suite,
so that became redundant. 

would you rather prefer to have validations in both sides?